add installers linux and winget (EN + PT) (#93)
ZupIT/docs-ritchie: content/en/Getting Started/Windows.md

## Step 2: Verify installation

> You can confirm if your installation went well by running this command:

Reviewer: ```suggestion
Confirm if your installation went well, run the command below:
```
